vochain/app.go

```diff
@@ -90,6 +90,7 @@ func NewBaseApplication(dbType, dbpath string) (*BaseApplication, error) {
 		TransactionHandler: transactionHandler,
 		blockCache:         lru.NewAtomic(32),
 		dataDir:            dbpath,
+		chainID:            "test",
```

Comment:
Are you sure that we want to hardcode the chainID value as “test” here? Even if it can be set later, seems strange.